sections/git_branch.zsh

```diff
@@ -22,9 +34,31 @@ spaceship_git_branch() {
   local git_current_branch="$vcs_info_msg_0_"
   [[ -z "$git_current_branch" ]] && return
 
+  # Show commit/tag info on detached HEAD
+  if [[ $SPACESHIP_GIT_BRANCH_SHOW_COMMIT == true ]] || [[ $SPACESHIP_GIT_BRANCH_SHOW_TAG == true ]]; then
+    local git_current_branch=$(git symbolic-ref HEAD --short 2> /dev/null)
```

Comment:
When on tags, This results in empty branch name with prefixed branch symbol.

![broken git tag](https://user-images.githubusercontent.com/10276208/40480498-5c52a816-5f6c-11e8-81fd-f636409b1a8f.png)

Reply:
You're right, @salmanulfarzy, but I have some doubts about `SPACESHIP_GIT_SYMBOL` role in the prompt as commented on https://github.com/denysdovhan/spaceship-prompt/pull/443#issuecomment-391826180: Is it used to advertise you're on a git repo?